### PR TITLE
fix outdated audit issues

### PR DESCRIPTION
**Pull request by trhinehart-godaddy**
npm update + fresh lock

### CODE REVIEW

**Comment by jgowdy** [comment]
Logo
**Checkmarx AST – Scan Summary & Details** – [6ebb94c1\-7172\-445a\-bd92\-70a3b38c4588](https://ast.checkmarx.net/projects/30afdbbe-cc9a-4c2a-aba7-72b2d948a69e/scans?id=6ebb94c1-7172-445a-bd92-70a3b38c4588&branch=update)

### Fixed Issues
Severity|Issue|File / Package|Scan Engine
---|---|---|---
HIGH|CVE\-2022\-2216|Npm\-parse\-url\-6\.0\.0|[CxSCA](https://ast.checkmarx.net/sca/#/projects/30afdbbe-cc9a-4c2a-aba7-72b2d948a69e/reports/6ebb94c1-7172-445a-bd92-70a3b38c4588/vulnerabilities/all)
HIGH|CVE\-2022\-2217|Npm\-parse\-url\-6\.0\.0|[CxSCA](https://ast.checkmarx.net/sca/#/projects/30afdbbe-cc9a-4c2a-aba7-72b2d948a69e/reports/6ebb94c1-7172-445a-bd92-70a3b38c4588/vulnerabilities/all)
HIGH|CVE\-2022\-2218|Npm\-parse\-url\-6\.0\.0|[CxSCA](https://ast.checkmarx.net/sca/#/projects/30afdbbe-cc9a-4c2a-aba7-72b2d948a69e/reports/6ebb94c1-7172-445a-bd92-70a3b38c4588/vulnerabilities/all)
HIGH|Cxc7705965\-e0f0|Npm\-@babel/core\-7\.17\.9|[CxSCA](https://ast.checkmarx.net/sca/#/projects/30afdbbe-cc9a-4c2a-aba7-72b2d948a69e/reports/6ebb94c1-7172-445a-bd92-70a3b38c4588/vulnerabilities/all)
MEDIUM|CVE\-2022\-0722|Npm\-parse\-url\-6\.0\.0|[CxSCA](https://ast.checkmarx.net/sca/#/projects/30afdbbe-cc9a-4c2a-aba7-72b2d948a69e/reports/6ebb94c1-7172-445a-bd92-70a3b38c4588/vulnerabilities/all)
MEDIUM|CVE\-2022\-25858|Npm\-terser\-4\.8\.0|[CxSCA](https://ast.checkmarx.net/sca/#/projects/30afdbbe-cc9a-4c2a-aba7-72b2d948a69e/reports/6ebb94c1-7172-445a-bd92-70a3b38c4588/vulnerabilities/all)
MEDIUM|CVE\-2022\-25858|Npm\-terser\-5\.12\.1|[CxSCA](https://ast.checkmarx.net/sca/#/projects/30afdbbe-cc9a-4c2a-aba7-72b2d948a69e/reports/6ebb94c1-7172-445a-bd92-70a3b38c4588/vulnerabilities/all)

**Review by asilvas-godaddy — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -13,35 +13,38 @@
...
     "nyc": "^15.1.0",
     "proxyquire": "^2.1.3",
     "rimraf": "^3.0.2",
-    "rollup": "^2.70.2",
+    "rollup": "^2.78.1",
     "rollup-plugin-terser": "^7.0.2",
     "webpack": "^4.46.0"
   },
   "ava": {
-    "babel": true,
```
Comment (trhinehart-godaddy) [inline package.json:44]: ava major removed babel support

File: scripts/clean.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const path = require('path');
+const { promisify } = require('util');
+const rimraf = promisify(require('rimraf'));
```
Comment (asilvas-godaddy) [inline scripts/clean.js:3]: no need for rimraf. node v14 supports recursive deletes. https://nodejs.org/dist/latest-v16.x/docs/api/fs.html#fspromisesrmpath-options

**Review by asilvas-godaddy — APPROVED** [review]
+1 with suggestion

File: scripts/clean.js (hunk 2 above)
Comment (trhinehart-godaddy) [reply scripts/clean.js:3]: already required by cli/npm scripts for windows vs linux compatibility, but also comes with glob support built in

**Pull request merged**